### PR TITLE
[ashell] Fix broken ashell build

### PR DESCRIPTION
**Pull request by jimmy-huang**
Signed-off-by: Jimmy Huang <jimmy.huang@intel.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ env:
     - ZEPHYR_GCC_VARIANT=zephyr
     - ZEPHYR_SDK_INSTALL_DIR=/opt/zephyr-sdk
     - ZEPHYR_BASE=$TRAVIS_BUILD_DIR/deps/zephyr
-    - ZEPHYR_SDK_VERSION=0.8.2
-    - ZEPHYR_SDK_DOWNLOAD_FOLDER=https://nexus.zephyrproject.org/content/repositories/releases/org/zephyrproject/zephyr-sdk/$ZEPHYR_SDK_VERSION-i686
-    - ZEPHYR_SDK_SETUP_BINARY=zephyr-sdk-$ZEPHYR_SDK_VERSION-i686-setup.run
+    - ZEPHYR_SDK_VERSION=0.9
+    - ZEPHYR_SDK_DOWNLOAD_FOLDER=https://nexus.zephyrproject.org/content/repositories/releases/org/zephyrproject/zephyr-sdk/$ZEPHYR_SDK_VERSION
+    - ZEPHYR_SDK_SETUP_BINARY=zephyr-sdk-$ZEPHYR_SDK_VERSION-setup.run
```
Comment (grgustaf) [inline .travis.yml]: I don't really want to make 0.9 the default while we have that lame debug build workaround.

File: scripts/analyze.sh [hunk 2]
```diff
@@ -329,7 +329,7 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
             echo "CONFIG_BMI160_TRIGGER_OWN_THREAD=y" >> arc/prj.conf.tmp
         fi
         bmi160=$(grep -E AmbientLightSensor $SCRIPT)
-        if [ $? -eq 0 ] || [ $DEV = "ashell" ]; then
+        if [[ $? -eq 0 ]] || [[ $DEV = "ashell" ]]; then
```
Comment (grgustaf) [inline scripts/analyze.sh]: I don't think these changes actually matter?

File: src/ashell/jerry-code.c [hunk 3]
```diff
@@ -13,7 +13,7 @@
 #include <ctype.h>
 
 /* JerryScript includes */
-#include "jerry-api.h"
+#include "jerryscript.h"
```
Comment (grgustaf) [inline src/ashell/jerry-code.c]: This is included by stuff below so doesn't really need to be there.

File: src/ashell/shell-state.c [hunk 4]
```diff
@@ -138,7 +138,7 @@ int32_t ashell_disk_usage(char *buf)
     ssize_t size = fs_size(file);
     fs_close(file);
 
-    printf("%5ld %s\n", size, filename);
+    printf("%d %s\n", size, filename);
```
Comment (grgustaf) [inline src/ashell/shell-state.c]: You dropped the 5 width specifier, and this won't build with 0.8.2 because of format issue. Also below.

**Comment by jimmy-huang** [comment]
dropping this since there's a another patch #878

**Pull request closed without merging**